azkaban-common/src/main/java/azkaban/executor/ExecutionRampDao.java

```diff
@@ -70,23 +71,19 @@ public ExecutableRampMap handle(final ResultSet resultSet) throws SQLException {
             ExecutableRamp.createInstance(
                 resultSet.getString(1),
                 resultSet.getString(2),
-                ExecutableRamp.Metadata.createInstance(
-                    resultSet.getInt(3),
-                    resultSet.getInt(4),
-                    resultSet.getBoolean(5)
-                ),
-                ExecutableRamp.State.createInstance(
-                    resultSet.getLong(6),
-                    resultSet.getLong(7),
-                    resultSet.getLong(8),
-                    resultSet.getInt(9),
-                    resultSet.getInt(10),
-                    resultSet.getInt(11),
-                    resultSet.getInt(12),
-                    resultSet.getBoolean(13),
-                    resultSet.getInt(14),
-                    resultSet.getBoolean(15)
-                )
+                resultSet.getInt(3),
+                resultSet.getInt(4),
+                resultSet.getBoolean(5),
+                resultSet.getLong(6),
+                resultSet.getLong(7),
+                resultSet.getLong(8),
+                resultSet.getInt(9),
+                resultSet.getInt(10),
+                resultSet.getInt(11),
+                resultSet.getInt(12),
+                resultSet.getBoolean(13),
+                resultSet.getInt(14),
+                resultSet.getBoolean(15)
```

Comment:
It is very difficult to follow which values are getting set in getInstance method as there are lot of values. Have you considered using builder pattern for this?

Reply:
Deepark had the same concern. I just tried to follow the existing pattern in Azkaban. As we both agreed, we can address this in a seperate PR.